Weather App/viewModel/WeatherForCityViewModel.swift

```diff
@@ -59,3 +56,9 @@ extension Notification.Name {
         return .init(rawValue: "WeatherForCityViewModel.weatherReturned")
     }
 }
+
+extension String {
```

Comment:
could move to outside the class to the separate folder with extensions to project?

Reply:
👍